### ISSUE TITLE
Update soil layer for drought phenology

### ISSUE THREAD

**Issue by jkshuman** [NGEET/fates#445]
Within the drought phenology routine (fates/biogechem/EdPhysoilogyMod.F90) the soil layer is set to the first soil layer, and should be set to the third soil layer per @rosiealice and phenology in CLM5. The first soil layer dries very quickly in CLM5.

```!Accumulate surface water memory of last 10 days.

    do i = 1,numWaterMem-1 !shift memory along one
       currentSite%water_memory(numWaterMem+1-i) = currentSite%water_memory(numWaterMem-i)
    enddo
    currentSite%water_memory(1) = bc_in%h2o_liqvol_sl(1)   !waterstate_inst%h2osoi_vol_col(coli,1)```

**Comment by jenniferholm** [comment]
@jkshuman - Checking in on this to see if it got wrapped into the new phenology updates that Ryan has been working on, or did this get forgotten? It seems like switching from 1 to 3 soil layers will make a (maybe big) difference and I didn't want to forget about this. 
How many soil layers do we have again in FATES?

**Comment by rgknox** [comment]
@jenniferholm  - I am going to add this to the phenology updates.  However, I would like to avoid using a specific layer index, and instead tie the soil water value to a real-world depth.  My plan is to use soil water from whichever layer is closest to the depth that currently matched layer 3 in the default layer scheme.
Another way to go about this would be to calculate the "center of depth" for the grid cell root-mass, and use that as the depth value.

**Comment by rgknox** [comment]
It looks like the default soil layer model in fates_next_api branch uses the '20SL_8.5m', which is 20 layers going down to 8.5 meters.  Here are the layer mid-points:

0.01
0.04
0.09  
0.16             
0.26
0.40
0.58
0.80
1.06
1.36
1.70
2.08
2.50
3.00
3.58
4.27
5.06
5.95
6.94
8.03

**Comment by rgknox** [comment]
I added a desired depth for drought phenology soil water comparisons. I set it to 10 cm.  FATES will use whatever layer is closest to 10cm as per the following commit:

https://github.com/NGEET/fates/pull/468/commits/3b14955ffb4c9774c8442a90a1c3ea0f29b5e901

**Comment by rgknox** [comment]
closing as is fixed, and will be further improved by #801
